docs/source/nlp/quantization.rst

```diff
@@ -133,15 +132,60 @@ Known issues
 * The supported and tested model family is Llama2. Quantizing other model types is experimental and may not be fully supported.
 
 
-Please refer to the following papers for more details on quantization techniques.
+Quantization-Aware Training (QAT)
+---------------------------------
 
-References
-----------
+QAT is the technique of fine-tuning a quantized model to recover model quality degradation due to quantization.
+While QAT requires much more compute resources than PTQ, it is highly effective in recovering model quality.
+To perform QAT on a calibrated model from PTQ, you need to further fine-tune the model on a downstream task using a small dataset before exporting to TensorRT-LLM.
+You can reuse your training pipeline for QAT.
+Since QAT is done after PTQ, the supported model families are the same as for PTQ.
+
+
+Example
+^^^^^^^
+
+The example below shows how to perform PTQ and QAT on a Supervised Finetuned Llama2 7B model to INT4 precision using tensor parallelism of 8 on 8x A5000 24GB GPUs.
```

Comment:
For ModelOpt OSS QAT, the minimum hardware is 1 x 80GB with activation checkpointing enabled. Can we reduce the compute requirement here as well using activation checkpointing?